src/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Regex.Timeout.cs

```diff
@@ -29,14 +29,19 @@ public partial class Regex
         // value as Timeout.InfiniteTimeSpan creating an implementation detail dependency only.
         public static readonly TimeSpan InfiniteMatchTimeout = Timeout.InfiniteTimeSpan;
 
-        // timeout for the execution of this regex
+        // timeout for the execution of this regex.
```

Comment:
Nit: it's strange to me to see a period added at the end, when it's not a full sentence and doesn't begin with a capital